src/bin/pg_autoctl/cli_enable_disable.c

```diff
@@ -501,6 +501,7 @@ cli_enable_maintenance(int argc, char **argv)
 			&(keeper.monitor),
 			keeper.config.formation,
 			keeper.config.groupId,
+			keeper.config.pgSetup.pgKind,
```

Comment:
Maybe we should just pass in the config here at this point, same for below.

Reply:
I think it's best to avoid having `monitor.c` depend on `keeper_config.c`.